test/integration/fn_minikube_config.go

```diff
@@ -0,0 +1,47 @@
+// +build integration
+
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"testing"
+)
+
+func testMinikubeConfig(t *testing.T) {
+	t.Parallel()
+	p := profileName(t)
```

Comment:
I wonder if this test will affect the rest of the tests running in parralell, if so, feel free to give it a different profile name and do the testing without starting it.


that brings a question, doest minikube write to the config file if it is not started?

Reply:
yup, it writes to the config even when stopped. What we can do here, is to actually set a default value, this wouldn't affect any other parallel test.